packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1004,7 +1004,8 @@ class StackViewport extends Viewport implements IStackViewport {
     const xVoxels = image.columns;
     const yVoxels = image.rows;
 
-    const zSpacing = image.sliceThickness || EPSILON;
+    // Note: For rendering purposes, we use the EPSILON as the z spacing.
```

Comment:
Could we add a tiny bit more info here?

Maybe something like "Note: For rendering purposes, we use the EPSILON as the z spacing. This is purely for internal implementation logic since we are still technically rendering 3D objects with vtk.js, but the asbtracted intention of the stack viewport is to render 2D images"

Or better wording if you can come up with it ^_^.